tenants/all/templates/labpulse-lfte.marko

```diff
@@ -1 +1 @@
-<common-standard-layout data=data />
+<common-standard-no-case-layout data=data />
```

Comment:
Does this need to be two separate layouts? Seems overkill to have all this split out for adding/removing a single block and the rest of the layout being identical. This can likely be config or prop driven (i.e `<common-standard-layout data=data case-of-week-block=false />` or something to that effect)